### DISCUSSION TITLE
do we have pre-built docker image ?

### DISCUSSION THREAD

**Discussion by AlwaysNoobCoder** [kubernetes/website#47010] [General]
I want to host kubernetes docs in our offline environment.

**Comment by sftim** [comment]
GitHub is not the right place for support requests.

If you're looking for help, check [Server Fault](https://serverfault.com/questions/tagged/kubernetes).

You can also post your question on the [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.
